components/org.wso2.carbon.identity.conditional.auth.functions.user/pom.xml

```diff
@@ -104,6 +104,19 @@
             <artifactId>nashorn-core</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.com.nimbusds</groupId>
+            <artifactId>nimbus-jose-jwt</artifactId>
```

Comment:
why are we adding these new dependencies/

Reply:
There were build issues due to this dependency not being present. There are multiple usages of this in the codebase however the dependency was missing. Therefore there were compilation issues.